services/storage/src/simcore_service_storage/handlers.py

```diff
@@ -379,9 +360,10 @@ async def create_folders_from_project(request: web.Request):
     )
 
 
+@routes.delete(f"/{api_vtag}/simcore-s3/folders/{{folder_id}}")  # type: ignore
 async def delete_folders_of_project(request: web.Request):
     folder_id = request.match_info["folder_id"]
-    user_id = request.query.get("user_id")
+    user_id = request.query["user_id"]
```

Comment:
is there a reason for this change?

Reply:
yes. The fields that are **required** i get them with ``[.]`` and those optional with ``.get`` by default.
Now, it is true that there is still NO validation of the request in place. 
I use this as a reminder so that in the coming PRs for the storage (see second point in #2776)